cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1294,9 +1294,10 @@
 
   <property>
     <name>messaging.system.topics</name>
-    <value>${audit.topic},${notification.topic}</value>
+    <value>${audit.topic},${metrics.topic.prefix}:${metrics.messaging.topic.num},${notification.topic}</value>
     <description>
-      A comma-separated list of topics that is always available in the system namespace
+      A comma-separated list of topics that is always available in the system namespace.
+      Topic prefix and topic partition are seperated by ":".
```

Comment:
Multiple topics sharing the same prefix and different with a numerical suffix can be specified with the syntax `<prefix>:<number>`.